Assets/Code/VehicleEditor/EditSystem.cs

```diff
@@ -307,4 +375,168 @@ partial class EditSystem : SystemBase {
     protected override void OnDestroy() {
         input.Disable();
     }
+
+    
+    void ShowPathToRoot(ref EntityCommandBuffer actionQueue , Entity parentPart)
+    {
+        // this is just some debug stuff to see how the parts where created
+        
+            Debug.Log("");
+            int iteration = SystemAPI.GetComponent<Part>(parentPart).layer; // ok idk if iteration is the right term but it will do just fine for now
+
+            int[] symmetryCounts = new int[iteration];
```

Comment:
should use data structures from `Unity.Collections` (e.g. `Unity.Collections.NativeArray<T>` (i think?)) - not that important for this use case or this prototype but reduces garbage collection overhead